projects/igniteui-angular/src/lib/grids/column.component.ts

```diff
@@ -836,6 +836,23 @@ export class IgxColumnComponent implements AfterContentInit {
 
         return pinnedCols.indexOf(this) === pinnedCols.length - 1;
     }
+    get gridRowSpan() {
+        return (this.rowEnd && this.rowEnd.indexOf('span') !== -1) ?
```

Comment:
parseInt excludes letters afaik, so you don't need the replace

Reply:
`parseInt('span 3', 10)` would be `NaN`